xcarchive/glob.go

```diff
@@ -0,0 +1,12 @@
+package xcarchive
+
+func escapeGlobPath(path string) string {
+	var escaped string
+	for _, ch := range path {
+		if ch == '[' || ch == '*' || ch == '?' || ch == '\\' {
```

Comment:
I think the `]` is missing from the list.

Reply:
if '[' is escaped ']' will not count as control character, based on https://golang.org/src/path/match.go.